src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -178,12 +192,30 @@ public void onTraversalStart() {
         // TODO if you change here, also change in CreateArrayIngestFiles
         // Get sample name
         final VCFHeader inputVCFHeader = getHeaderForVariants();
+        if (enableVCFHeaderProcessing) {
```

Comment:
this loop, as currently written, separates the header into an unnecessary number of chunks.  If you save up the command line headers until the end and write it along with the non-command line ones, we don't have as many separate groups

Reply:
Very happy that the uniqueness part has been sorted out!  I noticed that this loop is still doing the behavior where it  writes filtered out chunks when it sees them instead of grouping them together.  This is still creating multiple entries for the unique portion of the header, which is unnecessarily clogging the table.  This is the last request I have for this PR to tight it up

Reply:
The reason I kept the unique lines on their own in the DB is: if we want to only compare certain unique lines (e.g. just the Reblocking, not the HaplotypeCaller).